opennav_coverage/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.5)
 project(opennav_coverage)
 
 find_package(ament_cmake REQUIRED)
+find_package(ament_index_cpp REQUIRED)
```

Comment:
I don't see where this is used

Reply:
I added it because the tests were failing to building because of the line
```
#include "ament_index_cpp/get_package_share_directory.hpp"
```

It is in `test_server.cpp` and in `test_utils.cpp`